deso/index.go

```diff
@@ -24,171 +24,238 @@ const (
 
```

Comment:
Nit: Add a comment that PrefixUtxoOps is no longer used because we now pull UtxoOps from the node's BadgerDB.